pkg/occlient/occlient_test.go

```diff
@@ -613,9 +613,9 @@ func TestUpdateDCAnnotations(t *testing.T) {
 					t.Errorf("deployment Config annotations not matching with expected values, expected: %s, got %s", tt.annotations, updatedDc.Annotations)
 				}
 			} else if err == nil && tt.wantErr {
-				t.Errorf("test failed, expected: %s, got %s", "false", "true")
+				t.Error("error was expected, but no error was returned")
 			} else if err != nil && !tt.wantErr {
-				t.Errorf("test failed, expected: %s, got %s", "no error", "error")
+				t.Errorf("test failed, expected: %s, got %s", "no error", "error: "+err.Error())
```

Comment:
this is still wrong... "no error" should be replaced by `err` and `error` + err.Error() should be `tt.wantErr`

Example:

```go
t.Errorf("test failed, expected: %s, got %s", err, tt.wantErr)
```